master/internal/job/job.go

```diff
@@ -167,6 +167,7 @@ func (j *Jobs) Receive(ctx *actor.Context) error {
 		ctx.Respond(jobsInRM)
```

Comment:
not returning a response in command/command.go messages are fine I think. the actor system handles that case since we don't have any errors to report. right?